src/components/select/filterable-select/filterable-select.test.tsx

```diff
@@ -83,33 +220,1577 @@ test("combobox has correct accessible name when aria-labelledby prop is provided
+    </FilterableSelect>
+  );
+  screen.getByRole("combobox").focus();
+  await user.click(await screen.findByRole("option", { name: "green" }));
+
+  expect(onListScrollBottomFn).not.toHaveBeenCalled();
+});
+
+test("should apply the expected border radius styling to the list element", () => {
+  render(
+    <FilterableSelect
+      openOnFocus
+      label="filterable-select"
+      onChange={() => {}}
+      value=""
+    >
+      <Option value="opt1" text="green" />
+    </FilterableSelect>
+  );
+  screen.getByRole("combobox").focus();
+
+  expect(screen.getByTestId("select-list-wrapper")).toHaveStyleRule(
+    "border-radius",
+    "var(--borderRadius050)"
+  );
+});
+
+test("should apply the expected `maxWidth` styling when the prop is passed", () => {
+  render(
+    <FilterableSelect
+      label="filterable-select"
+      onChange={() => {}}
+      value=""
+      maxWidth="69%"
+    >
+      <Option value="opt1" text="green" />
+    </FilterableSelect>
+  );
+
+  expect(screen.getByTestId("input-presentation-container")).toHaveStyle(
+    "max-width: 69%"
+  );
+});
+
+test("should apply the expected `maxWidth` styling when the prop is not passed", () => {
+  render(
+    <FilterableSelect label="filterable-select" onChange={() => {}} value="">
+      <Option value="opt1" text="green" />
+    </FilterableSelect>
+  );
+
+  expect(screen.getByTestId("input-presentation-container")).toHaveStyle(
+    "max-width: 100%"
+  );
+});
+
+test("should show all options when `disableDefaultFiltering` is true and the user types in the input", async () => {
+  const user = userEvent.setup();
+  render(
+    <FilterableSelectWithState disableDefaultFiltering>
+      <Option value="opt1" text="a" />
+      <Option value="opt2" text="b" />
+      <Option value="opt3" text="c" />
+    </FilterableSelectWithState>
+  );
+  await user.type(screen.getByRole("combobox"), "a");
+
+  expect(await screen.findAllByRole("option")).toHaveLength(3);
+});
+
+test("should hide any non-matching options when `disableDefaultFiltering` is false and the user types in the input", async () => {
+  const user = userEvent.setup();
+  render(
+    <FilterableSelectWithState>
+      <Option value="opt1" text="a" />
+      <Option value="opt2" text="b" />
+      <Option value="opt3" text="c" />
+    </FilterableSelectWithState>
+  );
+  await user.type(screen.getByRole("combobox"), "a");
+
+  expect(await screen.findAllByRole("option")).toHaveLength(1);
+});
+
+test("should hide any non-matching options when `disableDefaultFiltering` is not set and the user types in the input", async () => {
+  const user = userEvent.setup();
+  render(
+    <FilterableSelectWithState>
+      <Option value="opt1" text="a" />
+      <Option value="opt2" text="b" />
+      <Option value="opt3" text="c" />
+    </FilterableSelectWithState>
+  );
+  await user.type(screen.getByRole("combobox"), "a");
+
+  expect(await screen.findAllByRole("option")).toHaveLength(1);
```

Comment:
**nitpick (non-blocking):** I see the test descriptions are different but this test looks identical to the previous one, could add `disableDefaultFiltering={false}` the the previous test to match the description a bit more accurately.

Reply:
yep the one above should be explicitly be setting the prop I'll update thanks